edge-modules/DirectMethodSender/Settings.cs

```diff
@@ -25,22 +26,38 @@ public class Settings
                     configuration.GetValue<string>("TargetModuleId", "DirectMethodReceiver"),
                     configuration.GetValue<TransportType>("TransportType", TransportType.Amqp_Tcp_Only),
                     configuration.GetValue<TimeSpan>("DirectMethodDelay", TimeSpan.FromSeconds(5)),
-                    Option.Maybe(configuration.GetValue<Uri>("AnalyzerUrl")));
+                    Option.Maybe(configuration.GetValue<Uri>("AnalyzerUrl")),
+                    configuration.GetValue("InvocationSource", "Local"),
+                    Option.Maybe<string>(configuration.GetValue<string>("ServiceClientConnectionString")));
```

Comment:
Make sure this parameter should be either defined with non-empty value or not defined.  Also don't define this parameter in settings.json with empty value.

Reply:
Yeah, this value is by default not-defined.